src/Library/demos/Memory Monitor/main.js

```diff
@@ -0,0 +1,27 @@
+import Adw from "gi://Adw";
+import Gio from "gi://Gio";
+
+const memory_monitor = Gio.MemoryMonitor.dup_default();
+
+const cache = new Map();
+cache.set("a", 1);
+cache.set("b", 2);
+cache.set("c", 3);
+
+memory_monitor.connect("low-memory-warning", (monitor, level) => {
+  if (level >= Gio.MemoryMonitorWarningLevel.LOW) {
```

Comment:
I think there should be a comment explaining why these are using a `>=` comparisons, since that's unexpected for an enumeration.